examples/lis2mdl_simpletest.py

```diff
@@ -5,10 +5,9 @@
 
 import time
 import board
-import busio
 import adafruit_lis2mdl
 
-i2c = busio.I2C(board.SCL, board.SDA)
+i2c = board.I2C()
```

Comment:
Same question.

Reply:
Changes in https://github.com/adafruit/Adafruit_CircuitPython_LIS2MDL/pull/13/commits/a5a8dfb3113f35f419452c2715b7ff99037e39c4